src/runtime/composables/useFormGroup.ts

```diff
@@ -44,7 +44,10 @@ export const useFormGroup = (inputProps?: InputProps, config?: any) => {
     return {
       inputId: computed(() => inputProps.id ?? formGroup?.inputId.value),
       name: computed(() => inputProps?.name ?? formGroup?.name.value),
-      size: computed(() => inputProps?.size ?? formGroup?.size.value ?? config?.default?.size),
+      size: computed(() => {
```

Comment:
```suggestion
      size: computed(() => inputProps?.size ?? config.size[formGroup?.size.value] ? formGroup?.size.value : config?.default?.size),
```
And it can return in a single line.